sdk/resourcemanager/Azure.ResourceManager.Core/src/Operation/PhVoidArmOperation.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary>
+    /// A calss representing an arm operation wrapper object.
+    /// </summary>
+    public class PhVoidArmOperation : ArmOperation
+    {
```

Comment:
same comments as above on 'Ph' prefix and modality

Reply:
this class is temporary